mix.exs

```diff
@@ -1,13 +1,13 @@
 defmodule GenRMQ.Mixfile do
   use Mix.Project
 
-  @version "3.0.0"
+  @version "3.0.1"
```

Comment:
```suggestion
  @version "4.0.0"
```
I think I want to actually bump the major version here. What do you think @mkorszun ?

Reply:
I agree, the lager / amqp / rabbitmq_common changes should drive this